sympy/assumptions/wrapper.py

```diff
@@ -126,6 +114,38 @@ def __new__(cls, expr, assumptions=None):
         obj.assumptions = assumptions
         return obj
 
+    _eval_is_algebraic = make_eval_method("algebraic")
```

Comment:
Maybe add a test to `sympy/assumptions/tests/test_wrapper` which tests if `AssumptionsWrapper` has all assumptions in `_assume_defined` defined, unless this is somehow already tested elsewhere.

Reply:
I've added this